docs/scripting/functions/PlayerTextDrawSetOutline.md

```diff
@@ -1,46 +1,50 @@
 ---
 title: PlayerTextDrawSetOutline
-description: Set the outline of a player-textdraw.
+description: Postavite outline za player-textdraw.
```

Comment:
This is the English file

Reply:
Ohhh shiit, my bad, I forgot about 'translations' directory